dolphinscheduler-dist/release-docs/LICENSE

```diff
@@ -445,6 +445,8 @@ The text of each license is also included at licenses/LICENSE-[project].txt.
     oshi-core 3.5.0: https://mvnrepository.com/artifact/com.github.oshi/oshi-core/3.5.0, EPL 1.0
     junit 4.12: https://mvnrepository.com/artifact/junit/junit/4.12, EPL 1.0
     h2-1.4.200 https://github.com/h2database/h2database/blob/master/LICENSE.txt, MPL 2.0 or EPL 1.0
+    mchange-commons-java 0.2.11: https://mvnrepository.com/artifact/com.mchange/mchange-commons-java/0.2.11, EPL 1.0
+    c3p0 0.9.5.2: https://mvnrepository.com/artifact/com.mchange/c3p0/0.9.5.2, EPL 1.0
```

Comment:
Hi,.I saw license[1] of c3p0 is LGPL or EPL,whether you can remove this jar?
[1]https://github.com/swaldman/c3p0/blob/c3p0-0.9.5.2/LICENSE

Reply:
hi, this is relatively difficult because Quartz relies on it. Type B licenses can also be introduced, but we need to be relatively cautious. What do you think?

Reply:
> hi, this is relatively difficult because Quartz relies on it. Type B licenses can also be introduced, but we need to be relatively cautious. What do you think?

I think we don't use directly that can be removed , take c3po as an example, we don't use c3p0, we use Druid as database connection pool,  so I think it could be removed, but I'm not sure, just try?

Reply:
I excluded c3p0 in the module service and api.Such as the following.
![image](https://user-images.githubusercontent.com/15651066/101143482-e9575e00-3651-11eb-95a6-67b48e07d6b6.png)

Reply:
@kevinsun2010 Sorry, I may have misled you, you can try as @lgcareer  said.

Reply:
> @kevinsun2010 Sorry, I may have misled you, you can try as @lgcareer said.

i see ,i will try.ths

Reply:
i try as @lgcareer  said. but  Compile-check  fail.  about +c3p0-0.9.5.2.jar
![2020-12-05_100700](https://user-images.githubusercontent.com/27861996/101230724-ea37d080-36e1-11eb-803d-98a38f79cec2.jpg)

Reply:
I commit some code to this PR，now it’s OK